cmd/server-main.go

```diff
@@ -119,8 +119,7 @@ func parseStorageEndpoints(eps []string) (endpoints []*url.URL, err error) {
 		if ep == "" {
 			return nil, errInvalidArgument
 		}
-		var u *url.URL
-		u, err = url.Parse(ep)
+		u, err := url.Parse(ep)
```

Comment:
This causes shadowing with "err error" return value. We need to keep it the same way as before or not return "err error" .